### ISSUE TITLE
Miniconda3 4.12.0 Windows installer doesn't support Add To Path

### ISSUE THREAD

**Issue by dpotapov** [ContinuumIO/anaconda-issues#12956]
We are using Miniconda3 installer on ephemeral CI/CD build machines and rely on tools being installed into the global %PATH%.

Since Miniconda3 4.12.0 the conda executable is no longer available in PATH despite the /AddToPath:1 had been provided via command-line flags to the installer (the option is documented here: https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html).

### Steps to Reproduce

1. Install Miniconda3 in silent mode with /AddToPath=1 flag

```
start /wait "" Miniconda3-latest-Windows-x86_64.exe /InstallationType=AllUsers /RegisterPython=1 /AddToPath=1 /S
```

2. Open new command line prompt
3. Type `conda -V`

### Expected Behavior

```
C:\>conda -V
conda 4.12.0

C:\>echo %PATH%
C:\ProgramData\Miniconda3;C:\ProgramData\Miniconda3\Library\mingw-w64\bin;C:\ProgramData\Miniconda3\Library\usr\bin;C:\ProgramData\Miniconda3\Library\bin;C:\ProgramData\Miniconda3\Scripts;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;....
```

### Actual Behavior

```
C:\>conda -V
'conda' is not recognized as an internal or external command,
operable program or batch file
```

The GUI installer has the option disabled:

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/5475708/172176710-ba7d79bf-e5c1-4b31-9894-32accd31e4b8.png">

It feels that this capability is disabled on purpose. Is there any reason for the change? What are future plans?

**Comment by pseudoyim** [comment]
Hello @dpotapov.

Yes, as of **[Anaconda Distribution 2022.05](https://docs.anaconda.com/anaconda/reference/release-notes/)** and **Miniconda-4.12.0**, we disabled the option to `AddToPath` when running the installation for `AllUsers`. We did this to address [CVE-2022-26526](https://nvd.nist.gov/vuln/detail/CVE-2022-26526), which describes a security risk that was found when allowing the system `PATH` to be edited when running the installation in `AllUsers` mode. The `AddToPath` _is_ still an option if you run the installation in `JustMe` mode.

We recommend the following workarounds:
- You could run the installation in `JustMe` mode.
- If you wish to continue using the `AllUsers` mode, you could remove the `/AddToPath=1` parameter from your command and add the installation path to your `PATH` in a separate step after the installation completes.
- Since the `AddToPath` for `AllUsers` option was disabled in Miniconda-4.12.0, you could just stick to installing an older version of Miniconda (e.g. v4.11.0) in order to keep using the option to `AddToPath` when installing for `AllUsers`. After the installation completes, you could then just run a `conda update conda`.

**Comment by dpotapov** [comment]
Thanks, @pseudoyim !

I'm considering this option:
> add the installation path to your PATH in a separate step after the installation completes.

As far as I can see there are conda.bat, conda.exe, activate.bat files either in C:\ProgramData\Miniconda3\condabin or C:\ProgramData\Miniconda3\Scripts. Which path is most appropriate to register in system PATH to let any user on a system do "conda activate"?

**Comment by pseudoyim** [comment]
Hello @dpotapov.

Apologies for the delay. I ran a test install with the last Miniconda that allowed for the AllUsers/AddToPath option (Miniconda3-4.11.0). Here's what I saw on the System %PATH% **before** the installation:

![Screen Shot 2022-06-15 at 10 32 18 AM](https://user-images.githubusercontent.com/9688260/173877941-3b0bec05-b8e3-4b21-92fb-1b4a0b8ec9a7.png)

And here's what I saw **after**:

![Screen Shot 2022-06-15 at 10 39 46 AM](https://user-images.githubusercontent.com/9688260/173878132-9bf7e81d-0383-4b80-9e68-8dd911fd7e7b.png)

So I believe you would want to add those paths that are outlined in the red rectangle.
